src/AspnetBoilerplateExtended.Application/User-Update-Details/IUserUpdateDetailsAppService.cs

```diff
@@ -0,0 +1,26 @@
+﻿using AspnetBoilerplateExtended.User_Update_Details.Dto;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AspnetBoilerplateExtended.User_Update_Details
+{
+    public interface IUserUpdateDetailsAppService
+    {
+        /// <summary>
+        /// Method for Getting User Details
+        /// </summary>
+        /// <param name="id">Current User Id</param>
+        /// <returns>Details of User</returns>
+        Task<UserUpdateDetailDto> GetUserDetailsAsync(string id);
+
+        /// <summary>
+        /// Method for Updating User Details
+        /// </summary>
+        /// <param name="input">UpdateDetailDto </param>
```

Comment:
Change variable name as per model and check for the same in all methods

Reply:
done